content/en/load-testing/Manual/900-create-report-cheatsheet.md

```diff
@@ -0,0 +1,50 @@
+---
+title: "Create Report Cheat-Sheet"
+linkTitle: "Cheat-Sheet Create_report"
```

Comment:
spelling: cheat sheet (no hyphen)
spelling: create_report (lowercase for command)
(both for title and linkTitle)